src/main/java/de/tum/in/www1/artemis/domain/PairwiseDistance.java

```diff
@@ -0,0 +1,77 @@
+package de.tum.in.www1.artemis.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.NaturalId;
+
+import java.io.Serializable;
+
+import javax.persistence.*;
+
+/**
+ *  Pairwise distance between two TextBlocks
+ */
+@Entity
+@Table(name = "pairwise_distance")
+public class PairwiseDistance implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @JsonIgnore
+    private Long id;
+
+    @Column(name = "block_i", nullable = false)
+    private long block_i;
+
+    @Column(name = "block_j", nullable = false)
+    private long block_j;
+
+    @Column(name = "distance", nullable = false)
+    private double distance;
+
+    @ManyToOne
+    @JsonIgnore
+    private TextExercise exercise;
+    // private ClusteringResult clusteringResult;
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public long getBlock_i() { return this.block_i; }
```

Comment:
Codacy found an issue: [The instance method name 'getBlock_i' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968857)